gst/nnstreamer/tensor_query/tensor_query_common.c

```diff
@@ -110,31 +110,29 @@ nnstreamer_query_connection_get_port (query_connection_handle connection)
   return conn->port;
 }
 
+
 /**
- * @brief Create requested socket.
+ * @brief Get socket address
  */
 static gboolean
-gst_tensor_query_socket_new (query_connection_handle conn_h,
-    GSocketAddress ** saddr)
+gst_tensor_query_get_saddr (const char *host, uint32_t port,
```

Comment:
In general, the TCP port field is 16 bits (i.e., int16_t). ```gst_tensor_query_get_saddr ``` also takes ```int16_t port``` for its second argument.

Reply:
Updated to uint16. Thanks.

Reply:
It is not ```uint16_t``` but ```int16_t```.

Reply:
Well..
`g_inet_socket_address_new` takes `guint16 port`. [link](https://www.freedesktop.org/software/gstreamer-sdk/data/docs/latest/gio/GInetSocketAddress.html#g-inet-socket-address-new)
Do we need negative port number?

Reply:
You are absolutely right! I am confused:(